### DISCUSSION TITLE
Roast Name/Scope Overlaying Chart -- Where can the text location be adjusted?

### DISCUSSION THREAD

**Discussion by bsmccle12** [artisan-roaster-scope/artisan#649] [Q&A]
Losing my mind!

![artisan](https://user-images.githubusercontent.com/87956778/126922892-036feced-fa94-4bad-9930-fdf2a2f965e2.JPG)

**Comment by poundy** [comment]
i don't think you can - it's meant to sit above the graphing area (from my use on Windows). What you have, to me, seems like a font/scaling issue. I can see you're using VNC and connecting to a RPi. Can you try when direct to the Pi and see if the behaviour is different. Can you also show us what is configured in your Config --> Curves --> UI page please, and consider attaching your settings file.

**Comment by CarefreeBuzzBuzz** [comment]
A lost mind is a good thing these days. Carpé Diem.   You didn’t tell us
enough but @ poundy had some thoughts.

On Sun, Jul 25, 2021 at 7:06 PM poundy ***@***.***> wrote:

> i don't think you can - it's meant to sit above the graphing area (from my
> use on Windows). What you have, to me, seems like a font/scaling issue. I
> can see you're using VNC and connecting to a RPi. Can you try when direct
> to the Pi and see if the behaviour is different. Can you also show us what
> is configured in your Config --> Curves --> UI page please, and consider
> attaching your settings file.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/artisan-roaster-scope/artisan/discussions/649#discussioncomment-1048950>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIW3J4AKREIWQJD4UXN7ZETTZS7KDANCNFSM5A7EP3XQ>
> .
>
-- 

__________________
Michael Herbert, Head Roaster
*Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com>*
Artisan RoasterScope <https://artisan-scope.org/>Team

**Reply by bsmccle12** [reply]
Sorry for the delay, it's been a hectic few weeks. Attached is a screenshot of the config.
![2021-08-16-203744_1920x1080_scrot](https://user-images.githubusercontent.com/87956778/129650421-a1c2c7ad-eaf4-4d5d-9703-23f12c006b37.png)

**Comment by MAKOMO** [comment]
Please upgrade to the latest version which is v2.4.6. Seems you run the outdated v2.4.4 Stretch build.
